WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerSetup.kt

```diff
@@ -27,6 +28,8 @@ data class MediaPickerSetup(
         intent.putExtra(KEY_CAMERA_ENABLED, cameraEnabled)
     }
 
+    fun isBrowser() = dataSource == DEVICE && allowedTypes.containsAll(MediaType.values().toList())
```

Comment:
I'd rather not have this logic here. Otherwise it will start behaving like the browser type class :-).

Reply:
😊...removed here 1de6b2a5a930a8e3d6578e4e2cfbe3c573bf0345